exec/java-exec/src/main/resources/bootstrap-storage-plugins.json

```diff
@@ -1,178 +1,178 @@
-          location: "/",
-          writable: false,
-          allowAccessOutsideWorkspace: false
+          "location": "/",
+          "writable": false,
+          "allowAccessOutsideWorkspace": false
         },
         "tmp" : {
-          location: "/tmp",
-          writable: true,
-          allowAccessOutsideWorkspace: false
+          "location": "/tmp",
+          "writable": true,
+          "allowAccessOutsideWorkspace": false
         }
       },
-      formats: {
+      "formats": {
         "psv" : {
-          type: "text",
-          extensions: [ "tbl" ],
-          delimiter: "|"
+          "type": "text",
+          "extensions": [ "tbl" ],
+          "delimiter": "|"
         },
         "csv" : {
-          type: "text",
-          extensions: [ "csv" ],
-          delimiter: ","
+          "type": "text",
+          "extensions": [ "csv" ],
+          "delimiter": ","
         },
         "tsv" : {
-          type: "text",
-          extensions: [ "tsv" ],
-          delimiter: "\t"
+          "type": "text",
+          "extensions": [ "tsv" ],
+          "delimiter": "\t"
         },
         "httpd" : {
-          type: "httpd",
-          logFormat: "%h %t \"%r\" %>s %b \"%{Referer}i\""
-          /* timestampFormat: "dd/MMM/yyyy:HH:mm:ss ZZ" */
+          "type": "httpd",
+          "logFormat": "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-agent}i\"",
+          "timestampFormat": "dd/MMM/yyyy:HH:mm:ss ZZ"
         },
         "parquet" : {
-          type: "parquet"
+          "type": "parquet"
         },
         "json" : {
-          type: "json",
-          extensions: [ "json" ]
+          "type": "json",
+          "extensions": [ "json" ]
         },
         "pcap" : {
-          type: "pcap"
+          "type": "pcap"
         },
         "pcapng" : {
-          type: "pcapng"
+          "type": "pcapng"
         },
         "avro" : {
-          type: "avro"
+          "type": "avro"
         },
         "sequencefile": {
-          type : "sequencefile",
-          extensions: [ "seq" ]
+          "type": "sequencefile",
+          "extensions": [ "seq" ]
         },
         "csvh" : {
-          type: "text",
-          extensions: [ "csvh" ],
-          delimiter: ",",
-          extractHeader: true
+          "type": "text",
+          "extensions": [ "csvh" ],
+          "delimiter": ",",
+          "extractHeader": true
         },
         "image" : {
-          type: "image",
-          extensions: [
+          "type": "image",
+          "extensions": [
             "jpg", "jpeg", "jpe", "tif", "tiff", "dng", "psd", "png", "bmp", "gif",
             "ico", "pcx", "wav", "wave", "avi", "webp", "mov", "mp4", "m4a", "m4p",
             "m4b", "m4r", "m4v", "3gp", "3g2", "eps", "epsf", "epsi", "ai", "arw",
             "crw", "cr2", "nef", "orf", "raf", "rw2", "rwl", "srw", "x3f"
           ]
         }
       },
-      enabled : true
+      "enabled" : true
     },
 
     s3: {
```

Comment:
Thanks a lot for wrapping field names with quotes. There are 4 (3 in s3 plugin and the last one is `cp.formats.csvh.extractHeader`) more left unwrap - could you wrap it as well, please?

Reply:
Fixed

Reply:
Nice. What about the 's3'? :)